inst/@sym/mpower.m

```diff
@@ -72,6 +72,8 @@
   end
 
   cmd = { 'x, y = _ins'
+          'if Version(sv) > Version("0.7.7.dev"):'
+          '    return x**y,'
```

Comment:
I'm guessing this leads to test failures.  Please do a different PR as it might get complicated for MatrixExpr...